### DISCUSSION TITLE
og:image for main page (index.html)

### DISCUSSION THREAD

**Discussion by elyserobinson** [adityatelange/hugo-PaperMod#743] [Q&A]
I'm trying to figure out what folder I need to put the main og:image in and where do I put the code for the image? The rest of the site for posts are pulling the images just fine.

**Comment by adityatelange** [comment]
https://github.com/adityatelange/hugo-PaperMod/blob/37f359ee2149abe147bba0a7dd4e1ff775dc2a56/layouts/partials/templates/opengraph.html#L22

**Reply by elyserobinson** [reply]
Will this change it for every post or just the main index.html page?

**Reply by adityatelange** [reply]
Wherever you don't have a cover image
